developer-guides/realtime-api/livechat-api/getinitialdata/README.md

```diff
@@ -10,24 +10,36 @@
 
 ```json
 {
-    "enabled": true,
-    "title": "Rocket.Chat",
-    "color": "#38B9E0",
-    "registrationForm": true,
-    "room": null,
-    "triggers": [],
-    "departments": [],
-    "allowSwitchingDepartments": true,
-    "online": true,
-    "offlineColor": "#666666",
-    "offlineMessage": "We are not online right now. Please leave us a message:",
-    "offlineSuccessMessage": "",
-    "offlineUnavailableMessage": "",
-    "displayOfflineForm": false,
-    "videoCall": false,
-    "offlineTitle": "Leave a message",
-    "language": "",
-    "transcript": false,
-    "transcriptMessage": "Would you like a copy of this chat emailed?"
+    enabled: true,
+    title: 'Rocket.Chat',
+    color: '#C1272D',
+    registrationForm: true,
+    room: null,
+    visitor: undefined,
+    triggers: [],
+    departments: [
```

Comment:
I suggest you keep the array of `departments` empty, not showing any data in our documentation.